source/_integrations/mealie.markdown

```diff
@@ -46,6 +47,10 @@ The integration will create a calendar for every type of meal plan:
 
 The integration will create a to-do list for every Mealie shopping list.
 
+## Sensors
+
+The integration provides sensors for the statistics like the number of recipes, categories, tags, tools and users.
```

Comment:
I think it could be helpful to have a list of all the sensor names with a description.

Reply:
It actually are those 5 haha

Reply:
Ah ok. Can you be a bit more descriptive? What are the categories, what are tags and tools?

Reply:
They are taxonomies within Mealie, so you have tools added to cook with, same with categories recipes are in.

They are also placed on the homepage of Mealie so I am assuming they are quite a well known name for the users.

cc @andrew-codechimp

Reply:
These taxonomies are deliberately vague within Mealie to allow people to adapt them to their needs.
As an example I use Categories for meal types and also things like baking, seasonings, etc. whilst Tags I use to indicate vegetarian and cuisine type, quick meals etc.
These should be considered a bit like Labels & Categories within HA, there's no one way to describe them, anyone using Mealie will know what they are.